budcamp/Homework1/Homework1/Program.cs

```diff
@@ -0,0 +1,277 @@
+﻿using System;
+
+namespace Homework1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //1.Exercise
+            int number = 24;
+            Console.WriteLine($"variable name is: {nameof(number)} and its value is: {number}");
+
+            //2.Exercise
+            Console.WriteLine("enter first num");
+            double input1 = int.Parse(Console.ReadLine());
+            Console.WriteLine("enter second num");
+            double input2 = int.Parse(Console.ReadLine());
+            Console.WriteLine("enter third num");
+            double input3 = int.Parse(Console.ReadLine());
+
+            double multiplication = input1 * input2 * input3;
+
+            Console.WriteLine($"The multiplication of your numbers is: {multiplication}");
+
+            //3.Exercise
+            Console.WriteLine("Enter your string");
+            string input = Console.ReadLine();
+
+            Console.WriteLine($"Result of swaping first and last chars is: {SwapFirstAndLastChar(input)}");
+
+            //4.Exercise
```

Comment:
wrong implementation
`4. Read two integers from the console. Calculate their sum. If values are equal, print triple of their sum, otherwise print sum divided by 8.`